.expeditor/config.yml

```diff
@@ -0,0 +1,33 @@
+# Documentation available at http://expeditor-docs.es.chef.io/
+
+product_key: marketplace
+
+slack:
+  notify_channel: supermarket-notify
+
+github:
```

Comment:
You can remove this block, everything is either the default value or deprecated.